tests/Doctrine/Tests/ORM/Functional/ExtraLazyCollectionTest.php

```diff
@@ -39,6 +39,10 @@ public function setUp()
         $class->associationMappings['phonenumbers']['fetch'] = ClassMetadataInfo::FETCH_EXTRA_LAZY;
         $class->associationMappings['phonenumbers']['indexBy'] = 'phonenumber';
 
+        unset($class->associationMappings['phonenumbers']['cache']);
+        unset($class->associationMappings['articles']['cache']);
+        unset($class->associationMappings['users']['cache']);
```

Comment:
You can't just `unset()` during `setUp()` and not add it back at `tearDown()`.
During the entire Doctrine test suite we modify existing entity mappings to suit the test needs.
If you turn it off, you may be suppressing other tests needs, so you remove in the beginning and add it back at the end. That's our rule. =)